direct/tpm2/combined_context_test.go

```diff
@@ -0,0 +1,117 @@
+package tpm2
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-tpm/direct/structures/tpm"
+	"github.com/google/go-tpm/direct/structures/tpm2b"
+	"github.com/google/go-tpm/direct/structures/tpma"
+	"github.com/google/go-tpm/direct/structures/tpml"
+	"github.com/google/go-tpm/direct/structures/tpms"
+	"github.com/google/go-tpm/direct/structures/tpmt"
+	"github.com/google/go-tpm/direct/structures/tpmu"
+	"github.com/google/go-tpm/direct/transport"
+	"github.com/google/go-tpm/direct/transport/simulator"
+)
+
+func ReadPublicName(t *testing.T, handle tpm.Handle, thetpm transport.TPM) tpm2b.Name {
+	readPublic := ReadPublic{
```

Comment:
Femto-nit: You can use [t.Helper()](https://pkg.go.dev/testing#T.Helper) here so that failures are printed with the line number of whatever test called this helper instead of the line number instead of the helper.

(I don't actually care too much about this one test, but for future reference this is a handy tool)

Reply:
Okay, I will be sure to keep this in mind for future test writing. Thank you.